Orange/widgets/evaluate/tests/test_owliftcurve.py

```diff
@@ -394,6 +394,12 @@ def test_points_from_results_precision_recall():
         np.testing.assert_almost_equal(res.respondents, respondents)
         np.testing.assert_almost_equal(res.thresholds, thresholds)
 
+    def test_area(self):
+        x = np.arange(5)
+        y = np.arange(5)[::-1]
+        area = compute_area(x, y)
+        self.assertEqual(area, 10)
+
```

Comment:
This test is not very strong - points start at 0, with equal space of `1`, the curve is symmetric. I can think of many functions that would pass this. :)

Here's a stronger test. With a nice ring to it, too. :)

```python
    def test_area(self):
        x = np.array([5, 8, 9, 11])
        y = np.array([7, 14, 8, 0])
        area = compute_area(x, y)
        self.assertEqual(area, 51)
```